### ISSUE TITLE
Option to search for genome accessions in search bar

### ISSUE THREAD

**Issue by MarikePalmer** [seq-code/registry#125]
As mentioned in discussion with Esraa Elsaeed on the Public Slack, it could be beneficial to search for genome accessions for taxa with validated names (and those that have been registered) to ensure that a named taxon under the SeqCode isn't named twice (i.e., a close match in NCBI to a genome with a generic taxon identifier may be in process or already validated in the registry, but the name isn't reflected in NCBI yet).

**Comment by lmrodriguezr** [comment]
To clarify: Is this about an option to search for genome accessions or an option to do genome search using something like ANI or a similar metric?

I think a genome accession search would be very useful indeed (or even a browsing). We already offer a JSON version of it ([see here](https://disc-genomics.uibk.ac.at/seqcode/type-genomes.json)), which is currently being used by both MiGA and GTDB, but I agree that a human-readable version would be great!

**Comment by MarikePalmer** [comment]
Yes, like a search bar where one can type in an accession and it brings up the species page if it exists for the accession. Because one can use MiGA or other genome-based searches to identify conspecific taxa, but actually searching in a human-readable version I think.
